wormhole-connect/src/routes/sdkv2/route.ts

```diff
@@ -95,13 +100,29 @@ export class SDKv2Route {
       return isSameToken(tokenId, fromTokenIdV2);
     });
 
-    const toTokenSupported = !!(
-      await this.rc.supportedDestinationTokens(
-        fromTokenIdV2,
-        fromChain.context,
-        toChain.context,
-      )
-    ).find((tokenId) => {
+    const supportedDestinationTokens = await this.rc.supportedDestinationTokens(
+      fromTokenIdV2,
+      fromChain.context,
+      toChain.context,
+    );
+
+    const toTokenSupported = !!supportedDestinationTokens.find((tokenId) => {
+      // TODO: SDKV2
+      // `tokenId.address` is a `UniversalAddress`, while `toTokenIdV2.address` is a `NativeAddress<C>`.
+      // To compare these two, we must convert the `UniversalAddress` to a `NativeAddress<C>`.
+      // In the case of Sui, this conversion requires looking up the address from the token bridge contract,
+      // which stores them in a map. Ideally, the SDK should provide a generic method for this conversion
+      // applicable to all chains.
+      // For routes involving the token bridge, there's only one supported destination token so
+      // we don't need to check the address.
+      if (
+        this.IS_TOKEN_BRIDGE_ROUTE &&
+        supportedDestinationTokens.length === 1 &&
+        toChain.chain === 'Sui' &&
+        toTokenIdV2.chain === 'Sui'
+      ) {
```

Comment:
I'm pretty sure this would return `true` for any given `destToken` if it's on Sui, which is incorrect

Reply:
This is an issue for any native Sui token

Reply:
Changing to check `SUI` token only